osbs/build/build_requestv2.py

```diff
@@ -22,12 +22,13 @@ class BuildRequestV2(BuildRequest):
     Wraps logic for creating build inputs
     """
 
-    def __init__(self, build_json_store, customize_conf=None):
+    def __init__(self, build_json_store, outer_template=None, customize_conf=None):
         """
         :param build_json_store: str, path to directory with JSON build files
```

Comment:
Can there be a matching `:param` comment?

        :param outer_template: str, path to outer template JSON

Reply:
will add